package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "calc",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "./node_modules/.bin/webpack-dev-server",
+    "build": "./node_modules/.bin/webpack -p",
+    "watch": "./node_modules/.bin/webpack --watch"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-core": "~6.26.0",
```

Comment:
Those `babel-*` packages are no longer maintained, I think we should use packages that start with `@babel/*`